test-data/unit/check-attr.test

```diff
@@ -1133,6 +1148,27 @@ A(None, None)
 
 [builtins fixtures/attr.pyi]
 
+[case testAttrsOptionalConverterNewPackage]
+# flags: --strict-optional
```

Comment:
Is it not on by default for tests? I tried to check but either nobody else is using this or github search is broken.

Reply:
Not sure, I just copied it from the test above.

Reply:
Ah okay.